src/docfx/build/link/LinkResolver.cs

```diff
@@ -116,7 +127,7 @@ internal class LinkResolver
         public IEnumerable<FilePath> GetAdditionalResources() => _additionalResources.Value;
 
         private (Error? error, string href, string? fragment, LinkType linkType, FilePath? file, bool isCrossReference) TryResolveAbsoluteLink(
-            SourceInfo<string> href, FilePath hrefRelativeTo, FilePath inclusionRoot)
+            SourceInfo<string> href, FilePath hrefRelativeTo, FilePath inclusionRoot, LinkInfo? linkInfo)
```

Comment:
Keep markdown related code (LinkInfo to LinkNode transformation) in MarkodwnEngine
```suggestion
            SourceInfo<string> href, FilePath hrefRelativeTo, FilePath inclusionRoot, LinkNode? linkNode)
```

Reply:
Done, create a method transformLinkInfo to handle LinkInfo to LinkNode transformation.